test/crossbrowser/src/browser/events.ts

```diff
@@ -0,0 +1,204 @@
+import DMP from '../../../lib/sdkdriver/src/dmp';
+
+import {
+  add,
+  remove
+} from './resources';
+
+import {
+  serializeLocalParticipant,
+  serializeLocalTrack,
+  serializeLocalTrackPublication,
+  serializeParticipant,
+  serializeRemoteTrack,
+  serializeRoom
+} from './serialize';
+
+/**
+ * Send {@link LocalParticipant} events to the {@link SDKDriver}.
+ * @param {DMP} dmp
+ * @param {LocalParticipant} localParticipant
+ */
+function sendLocalParticipantEvents(dmp: DMP, localParticipant: any): void {
+  localParticipant.on('trackAdded', (track: any) => {
+    dmp.sendEvent({
+      args: [serializeLocalTrack(track)],
+      source: serializeLocalParticipant(localParticipant),
```

Comment:
I guess I though this would be the resource ID? But I guess there is a benefit to sending full state here in case anything changed?